modules/swagger-codegen/src/main/resources/akka-scala/README.mustache

```diff
@@ -1,53 +1,41 @@
-# NAME
+# {{artifactId}}
 
 {{appName}}
 
 {{#appDescription}}{{{appDescription}}}{{/appDescription}}
 
-# VERSION
+## Requirements
 
-Automatically generated by the [Swagger Codegen](https://github.com/swagger-api/swagger-codegen) project:
-
-- API version: {{appVersion}}
-- Package version: {{moduleVersion}}
-{{^hideGenerationTimestamp}}
-- Build date: {{generatedDate}}
-{{/hideGenerationTimestamp}}
-- Build package: {{generatorClass}}
-{{#infoUrl}}
-For more information, please visit [{{{infoUrl}}}]({{{infoUrl}}})
-{{/infoUrl}}
-
-# Requirements
-
-Building the API client library requires [Maven](https://maven.apache.org/) to be installed.
```

Comment:
Understood that you don't need these section in the auto-generated README but can we still keep this while you maintain a customized version of the templates to meet your unique requirement (e.g. putting Huawei logo in the README)?

This part is pretty standard across all the auto-generated README.

Reply:
Not only need maven but also need jdk and other `Requirements `.
Here I add a list of tools.

Reply:
Sorry I wasn't clear. I'm referring to line 9 to 21

Reply:
`Automatically generated by the ...` such doc not exist in the Java readme.  `Build date` and `API version` can be kept. We keep the same with Java readme.

Reply:
fixed. move brief version info to project section.